src/data/types/types.jsx

```diff
@@ -1,14 +1,18 @@
 /* eslint-disable import/prefer-default-export */
-export const GET_REVIEW_META_DATA = 'GET_REVIEW_META_DATA';
-export const GET_REVIEW_META_DATA_SUCCESS = 'GET_REVIEW_META_DATA_SUCCESS';
-export const UPDATE_AVG_RATING = 'UPDATE_AVG_RATING';
+// PRODUCT DATA TYPES
 export const GET_PRODUCT_DATA = 'GET_PRODUCT_DATA';
 export const GET_PRODUCT_DATA_SUCCESS = 'GET_PRODUCT_DATA_SUCCESS';
-export const GET_PRODUCTS_LIST = 'GET_PRODUCTS_LIST';
-export const GET_PRODUCTS_LIST_SUCCESS = 'GET_PRODUCTS_LIST_SUCCESS';
 export const GET_PRODUCT_STYLES_SUCCESS = 'GET_PRODUCT_STYLES_SUCCESS';
 export const UPDATE_SELECTED_STYLE = 'UPDATE_SELECTED_STYLE';
+export const UPDATE_SELECTED_SIZE = 'UPDATE_SELECTED_SIZE';
+export const UPDATE_SELECTED_QUANTITY = 'UPDATE_SELECTED_QUANTITY';
+export const UPDATE_CART = 'UPDATE_CART';
 
+// PRODUCT LIST TYPES
+export const GET_PRODUCTS_LIST = 'GET_PRODUCTS_LIST';
+export const GET_PRODUCTS_LIST_SUCCESS = 'GET_PRODUCTS_LIST_SUCCESS';
 
-// Reviews
-// export const GET
+// REVIEW TYPES
+export const GET_REVIEW_META_DATA = 'GET_REVIEW_META_DATA';
+export const GET_REVIEW_META_DATA_SUCCESS = 'GET_REVIEW_META_DATA_SUCCESS';
+export const UPDATE_AVG_RATING = 'UPDATE_AVG_RATING';
```

Comment:
Reorganized types for better readability. This might have a merge conflict with the master branch since I moved around some of the types. But these changes should be accepted over the previous.